WORKSPACE

```diff
@@ -88,3 +88,16 @@ http_archive(
         "https://github.com/nlohmann/json/releases/download/v3.6.1/include.zip",
     ],
 )
+
+# C++ Prometheus Client library.
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
+
+http_archive(
+    name = "com_github_jupp0r_prometheus_cpp",
+    strip_prefix = "prometheus-cpp-master",
```

Comment:
Please pin to a version (and add sha256), for consistency with other deps.

Reply:
Sure. Where can I get SHA256 of this dependency repo, or do I generate a random one with some rules?

Reply:
pin to v0.9.0, and sha256 added